app/models/concerns/account_interactions.rb

```diff
@@ -80,9 +80,12 @@ def follow_mapping(query, field)
     has_many :active_relationships,  class_name: 'Follow', foreign_key: 'account_id',        dependent: :destroy
     has_many :passive_relationships, class_name: 'Follow', foreign_key: 'target_account_id', dependent: :destroy
 
-    has_many :following, -> { order('follows.id desc') }, through: :active_relationships,  source: :target_account
+    has_many :following, -> { order('follows.id desc') }, through: :active_relationships, source: :target_account
     has_many :followers, -> { order('follows.id desc') }, through: :passive_relationships, source: :account
 
+    has_many :tag_relationships, class_name: 'TagFollow', foreign_key: 'account_id', dependent: :destroy
```

Comment:
I think the name twist makes more sense with `active_relationships` and `passive_relationships` because we have Account on both ends.
With the tags it is unidirectional, so IMHO such a naming twist is not needed.